lib/ui/semantics.dart

```diff
@@ -488,33 +488,39 @@ class SemanticsUpdateBuilder extends NativeFieldWrapperClass2 {
     String increasedValue,
     String decreasedValue,
     TextDirection textDirection,
-    int hitTestPosition,
     Float64List transform,
-    Int32List children,
+    // TODO(yjbanov): remove after moving the framework to the new param names.
+    @deprecated Int32List children,
+    Int32List childrenInTraversalOrder,
```

Comment:
If both are identical (as I expect they are in most cases), should we maybe only transfer the list once?

Reply:
Maybe not worth the overhead as the lists are usually small and just integers?

Reply:
I wouldn't worry about performance for now. Our semantics tree is very sparse compared to the render tree. And you are right, `Int32List` is a tightly laid out chunk of memory filled with 32-bit integers.